api-reference/endpoint/query.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: "Check Software EOL"
+api: "POST https://eol.xeol.com/api/v1/query"
+description: "This endpoint checks if the provided software identifier(s) are end-of-life"
+---
+
+### Body
+
+<ParamField body="identifiers" type="object[]" required="true">
+  list of software identifiers to check for end-of-life status
+
+  <Expandable title="identifier">
+    <ResponseField name="type" type="string">
+      Identifier type for this software.
+      
+      Can be either `cpe` `purl` `custom`
+    </ResponseField>
+
+    <ResponseField name="identifier" type="string">
+      Unique identifier string for the software.
+      
+      If custom, just use the software's name like this: `redis`
+    </ResponseField>
+
+    <ResponseField name="version" type="string">
+      Version for this software
+    </ResponseField>
+
+    <ResponseField name="lts" type="boolean">
```

Comment:
`lts` is in the request body here, but we don't actually use `lts` information for matching at all. 

For example, Alpine 8 might be an LTS release, but Alpine 9 is not.

Would searching for this return Alpine 9 correctly? 
{
"type": "os",
"identifier": "cpe:/alpine"
"version": "9"
"lts": true,
"els": "xxx",
}